### PR TITLE
add some chinese language version docs

### PR DESCRIPTION
**Pull request by Rackar**

### CODE REVIEW

**Comment by avelino** [comment]
@diecoz you understand a little Chinese, can you revise this PR pls

**Comment by Rackar** [comment]
hello?

**Comment by avelino** [comment]
Sorry, but I am not cover to review in your contribution, do not understand anything Chinese :(

**Comment by Rackar** [comment]
Sure. Could you accept it, and wait for issues from Chinese?  We donot know some languages, So need to translate.

File: content/case-studies/_index.cn.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+title: "案例"
+date: 2018-03-01T13:59:08-03:00
+weight: 15
+menu: main
+---
+
+公司和/或项目使用perst？ 描述您的用例 [这里](https://github.com/prest/prest/issues/new?title=Case%20Studies:%20).
```
Comment (cassiobotaro) [inline content/case-studies/_index.cn.md]:
```suggestion
公司和/或项目使用prest？ 描述您的用例 [这里](https://github.com/prest/prest/issues/new?title=Case%20Studies:%20).
```

File: content/batch-operations/_index.cn.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+```
+[
+    {"FIELD1": "string value", "FIELD2": 1234567890},
+    {"FIELD1": "other string value", "FIELD2":1234567891},
+]
+```
+
+默认的insert方法使用多个元组值，例如`insert into table values（“value”，123），（“other”，456）`。 返回插入的行。
+您可以使用带有值`copy`的`Perst-Batch-Method`头来更改此行为。 它对大量插入很有用，但返回为空。
```
Comment (cassiobotaro) [inline content/batch-operations/_index.cn.md]:
```suggestion
您可以使用带有值`copy`的`Prest-Batch-Method`头来更改此行为。 它对大量插入很有用，但返回为空。
```

**Comment by Rackar** [comment]
> SGTM, @Rackar need your dedication to keep the documentation in Chinese

OK

**Comment by avelino** [comment]
Thanks @Rackar

**Pull request merged**